puffin_http/examples/server.rs

```diff
@@ -15,7 +15,8 @@ fn main() {
     let mut frame_counter = 0;
 
     loop {
-        puffin::profile_scope!("main_loop");
+        let data = format!("frame {}", frame_counter);
+        puffin::profile_scope!("main_loop", data);
```

Comment:
```suggestion
        puffin::profile_scope!("main_loop", format!("frame {}", frame_counter));
```

Reply:
Changes are ok for me, do you prefer I update the pull request, or directly commit the suggestion ?

For final, I prefer to have only one commit, with directly commit it can be squashed and fast forward merge ?

Reply:
Can squish on merge.